lidar_bev:
  ros__parameters:
    cloud_topic: "/wamv/sensors/lidars/ouster_lidar_sensor/points"
    lidar_tf_frame: "/wamv/wamv/base_link/ouster_lidar_sensor"
    camera_tf_frame: "/stereo_camera"
    camera_fov: "110.0"
    intensity_threshold: "0.05"
    cell_size: "0.05"
    cell_size_height_map: "0.25"
    max_height: "3.0"
    num_slices: "1"
    grid_dim: "70"
    grid_dim_height_map: "300"
    height_threshold: "0.10"
    crop_180: "true"
    remove_floor: "false"
    planes: "32"
    h_res: "0.2"
    v_res: "1.33"
    low_opening: "-30.67"
    max_expected_intensity: "30"